packages/List/src/List.stories.tsx

```diff
@@ -223,10 +223,10 @@ const listWithAction: OptionType[] = [
   },
   {
     type: 'list',
-    label: 'Discussing progress on individual goals',
+    label: 'Discussing progress on individual goals long titleeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
```

Comment:
Is it necessary to keep this long title in a story?

Reply:
It's so we can test if it's wrapping correctly in the list story. Maybe we don't need to have it anymore... I don't find either way.